Framework/Crystal/src/FindSXPeaksHelper.cpp

```diff
@@ -15,7 +15,10 @@
 
 #include "MantidTypes/SpectrumDefinition.h"
 
+#define BOOST_ALLOW_DEPRECATED_HEADERS
```

Comment:
Same comment as previous about moving to cmakelists defintion.

Reply:
Same response as previous